src/embind/embind.js

```diff
@@ -44,6 +44,9 @@ var LibraryEmbind = {
     Module['getLiveInheritedInstances'] = getLiveInheritedInstances;
     Module['flushPendingDeletes'] = flushPendingDeletes;
     Module['setDelayFunction'] = setDelayFunction;
+#if NO_DYNAMIC_EXECUTION
+    Module['NO_DYNAMIC_EXECUTION'] = true;
```

Comment:
is this just for the names? or is it necessary for functionality?

Reply:
Is there no way to figure it out in the tests themselves?

if not, we can use `IN_TEST_HARNESS` which is defined only when we are testing. I mean,
````
#if NO_DYNAMIC_EXECUTION
#if IN_TEST_HARNESS
   // do that thing
````